script.js

```diff
@@ -9,11 +9,64 @@ let sprite = {
 function getHungry() {
     if (sprite.hunger < 50) {
         hungerMoodle.classList.add("active")
+    } else{
+        hungerMoodle.classList.remove("active")
+
     }
 
     sprite.hunger--
 
-    console.log(sprite.hunger)
+    console.log(sprite)
```

Comment:
Do you still need this console log? 🤔 If not, remove it and add another commit. Total nitpick but good to do so you can go through the full comment, update, push again process